packages/web-runtime/src/App.vue

```diff
@@ -79,6 +80,33 @@ export default defineComponent({
 
     const activeRoute = computed(() => router.resolve(unref(router.currentRoute)))
 
+    const modalComponent = ref<VNodeRef>()
+    const onModalConfirm = (...args) => {
+      console.log(111, args)
```

Comment:
👀

Reply:
I've enabled auto-merge for this PR now. Just for the thrill of finding more dev leftovers and then having to be fast enough with pushing again 😁